crates/issue/src/lib.rs

```diff
@@ -246,7 +247,7 @@ impl<T: Config> Module<T> {
     /// Requests CBA issuance, returns unique tracking ID.
     fn _request_issue(
         requester: T::AccountId,
-        amount_polkabtc: PolkaBTC<T>,
+        amount_btc: PolkaBTC<T>,
```

Comment:
maybe rename to `amount_requested`. Then we can slowly get rid of the btc and polkabtc references in the var names.